picard/ui/ui_options_renaming.py

```diff
@@ -173,7 +160,7 @@ def retranslateUi(self, RenamingOptionsPage):
 "<html><head><meta name=\"qrichtext\" content=\"1\" /><style type=\"text/css\">\n"
```

Comment:
This line seems to have the only call to _translate in all picard's code (at least from a quick grep I did). Maybe it can be changed to _ and remove the _translate entry from _translate_re in setup.py ?